library/kotlin/src/io/envoyproxy/envoymobile/BUILD

```diff
@@ -48,3 +48,32 @@ envoy_mobile_kt_library(
         "//library/java/src/io/envoyproxy/envoymobile/engine:envoy_base_engine_lib",
     ],
 )
+
+kt_jvm_library(
+    name = "envoy_kt_jvm_lib",
```

Comment:
Is unable to find `"//library/java/src/io/envoyproxy/envoymobile/engine/types:envoy_c_types_lib"`

Also tried using the `kt_jvm_library` rule